README.md

```diff
@@ -149,56 +158,294 @@ This application will prompt users to fill in three or four games that they like
-To add an additional song:
+    nano config.env
+Set credentials - replace the xxx here too :
 
-`python run.py ingest --engine_string=<engine_string> --artist=<ARTIST> --title=<TITLE> --album=<ALBUM>`
+    AWS_SECRET_KEY=xxxxxxxxxxxxxxxxx
+    AWS_ACCESS_KEY=xxxxxxxxxx
 
-By default, `python run.py ingest` adds *Minor Cause* by Emancipator to the SQLite database located in `sqlite:///data/tracks.db`.
+
+### 4.Set configurations:
+Go to config.py in the config directory:
+
+    cd config/ 
+    
+ Within `config.py` Please change the following bucket name with your own bucket name.: 
+ 
+    S3_BUCKET_NAME = 'nw-adityagudal-s3'
+    
+To create local SQLlite schema let the variable `local` be as is, currently it is:
+    
+    local=True
+ 
+To create RDS instance change the variable `local`:
+     
+     local=False
+
+### 5. Store data into S3
+
+First check step 3 and 4, make sure your:
+   * Environment variables are set.
+   * Requirements and dependencies are met.
+   * Set configurations in config.py
+   
+Set your bucket name in `config.py` within config directory.
+
+Go to the root directory and Run the following:
+
+  * Build a docker image with tag games:
+  
+  		docker build -t games .
+
+   * Run the docker container:
+   
+    docker run --env-file=config/config.env games run.py
+
+    
+Note this `run.py` depends on `src/store_data_in_s3.py`
+
+This should store your data into the S3 bucket.
+
+### 6. Initialize the database 
+
+#### Create the database with tables
+ First check step 3 and 4, make sure your:
+   * Environment variables are set.
+   * Requirements and dependencies are met.
+   * Set configurations in config.py
+ 
+To create the database in the location configured in `config.py` with tables customer,metacritic,sales and steam, run: 
+
+To create a database locally use make sure to set `local=True` in `config.py`:
+
+Note `create_db.py` depends on `src/clean_data.py` `src/helpers.py` and `src/game_database.py`
+
+Go to the root directory and run:
+
+    python create_db.py
+
+By default, `python create_db.py ` with `local=True` creates a database at `sqlite:///data/games.db` with the tables customer, metacritic, sales and steam.
 
 #### Defining your engine string 
 A SQLAlchemy database connection is defined by a string with the following format:
 
 `dialect+driver://username:password@host:port/database`
 
-The `+dialect` is optional and if not provided, a default is used. For a more detailed description of what `dialect` and `driver` are and how a connection is made, you can see the documentation [here](https://docs.sqlalchemy.org/en/13/core/engines.html). We will cover SQLAlchemy and connection strings in the SQLAlchemy lab session on 
+The `+dialect` is optional and if not provided, a default is used. For a more detailed description of what `dialect` and `driver` are and how a connection is made, you can see the documentation [here](https://docs.sqlalchemy.org/en/13/core/engines.html). 
```

Comment:
it is good to provide all this background info, but if it is not necessary, i would advise against it. too much text distracts the user from what actions he/she has to take. imo, a readme.md should contain the minimum necessary instructions to get the application up and running,

Reply:
Oh yes, I will remove this. Thanks for pointing out!